### ISSUE TITLE
Details on the Object Detection Training for BDD100k

### ISSUE THREAD

**Issue by ukdsvl** [zhengziqiang/ForkGAN#11]
Hi,

Interesting paper! Can you please elaborate upon:

1. If the new 2020 labels of BDD100k were used for object detection training, or the old ones ?
2. In the paper, you have reported results for 9 classes of BDD100k (person, rider, car, bus, truck, bike, motor, traffic light, traffic sign). Does it mean you haven't used the 10th class "train" at al, or perhaps it was used, but only the results not reported ?
3. What is the hyper-parameter configuration for training the object detector, along with the number of iterations/ epochs for training the same ?

**Comment by ukdsvl** [comment]
Gentle reminder for addressing the same.

Thanks and Regards,

**Comment by ukdsvl** [comment]
would you please comment ?
